client/extensions/woocommerce/app/order/create.js

```diff
@@ -50,8 +52,18 @@ class Order extends Component {
 
 	// Saves changes to the remote site via API
 	saveOrder = () => {
-		const { siteId, order } = this.props;
-		this.props.saveOrder( siteId, order );
+		const { site, siteId, order, translate } = this.props;
+		const onSuccess = ( dispatch, orderId ) => {
```

Comment:
Should the messaging here be contextual. Like when creating a new order, should the messaging say "Order created." / "Unable to create order."? I think the current verbiage works in both cases, but figured it would be worth asking :)